### ISSUE TITLE
Download Button won't work - after Update to CiviCRM 5.13.4

### ISSUE THREAD

**Issue by Detsieber** [systopia/de.systopia.donrec#80]
I don't know the exact version, when it broke... But with CiviCRM 5.13.4 it is not possible anymore to download (again) previously created donation receipts.

Error message: 

```
Supplied mime-type is not accepted

CRM_Core_Exception: Supplied mime-type is not accepted in /html/sites/all/modules/civicrm/CRM/Core/Invoke.php on line 311
--
Exception trace
# | Function | Location
0 | CRM_Core_Page_File->run(Array, null) | /html/sites/all/modules/civicrm/CRM/Core/Invoke.php:311
1 | CRM_Core_Invoke::runItem(Array) | /html/sites/all/modules/civicrm/CRM/Core/Invoke.php:85
2 | CRM_Core_Invoke::_invoke(Array) | /html/sites/all/modules/civicrm/CRM/Core/Invoke.php:52
3 | CRM_Core_Invoke::invoke(Array) | /html/sites/all/modules/civicrm/drupal/civicrm.module:444
4 | civicrm_invoke('file') | /html/includes/menu.inc:527
5 | menu_execute_active_handler() | /html/index.php:21
6 | {main} |  
```

**Comment by bjendres** [comment]
I would assume this is due to the last security release, see https://civicrm.org/advisory/civi-sa-2019-15-xss-via-forged-mime-type. It might help to just add zip files to the new ``requestableMimeTypes`` setting via the API explorer:
```
Settings.create requestableMimeTypes=application/zip,image/jpeg,image/pjpeg,image/gif,image/x-png,image/png,image/jpg,text/html,application/pdf
```
@Detsieber Could you check if that works for you?

**Comment by Detsieber** [comment]
Unfortunately, it doesn't work.

I also looked into the file sites/all/modules/civicrm/CRM/Core/Page/File.php where the error is thrown, and I think that line 75 won't ever work... ($mimeType is empty, and passing it to any function won't fill it - maybe something wrong with that line?)

**Comment by jensschuppe** [comment]
Released with 1.7
